nmtwizard/framework.py

```diff
@@ -477,6 +479,20 @@ def translate_fn(*args, **kwargs):
             else:
                 return self.trans(*args, **kwargs)
 
+        def gzip_unzip_file(path_input, flag):
```

Comment:
Not sure about the benefit of the `flag` argument. Just define 2 functions: `compress_file` and `decompress_file` and put these at the end of the file.